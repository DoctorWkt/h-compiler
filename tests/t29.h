int a=2, *fred, jim[20], pizza[]= {1,2,3};
char kim, *dani, thing[]={'a','b','c'}, mesg[]="A message";

void strcpy(char *dst, char *src); 

void main(int argc, char *argv)
{
  int x= a + 3, y = x + 2;
  printn(a);
  printn(x);
  printn(y);
}
