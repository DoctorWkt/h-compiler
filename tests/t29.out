000002
000005
000007
